### ISSUE TITLE
onCancel event not working

### ISSUE THREAD

**Issue by folamy** [somteacodes/vue3-paystack#4]
Hello @somteacodes! First I would like to commend your effort, well done for this plugin.
Everything works fine, but I notice the cancel event does not get triggered when you cancel or close a payment modal!

**Comment by tunjixy** [comment]
Yeah @somteacodes everything works fine but the onCancel event doesn't get triggered when closing the paystack modal

**Comment by tunjixy** [comment]
Yeah @somteacodes everything works fine but the onCancel event doesn't get triggered when closing the paystack modal

**Comment by lanna-code00** [comment]
Hey guys @tunjixy  @somteacodes  @folamy , I need help oo

https://github.com/somteacodes/vue3-paystack/issues/11
This issue is what I can't resolve. But it works for public key.

I'm using the same secret key in my backend and it works, i don't know I keep getting this error when I use in the frontend

**Comment by bretoreta** [comment]
Hey @somteacodes any updates with regards to this issue? We really love your package and we are using it in production! We wouln't want aomething to break.

**Comment by somteacodes** [comment]
The issue has now been fixed
